docs/getting-started-guides/azure.md

```diff
@@ -0,0 +1,104 @@
+---
+---
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+1. An Azure Subscription. If you don't already have one, you may create one on [azure.microsoft.com](https://azure.microsoft.com).
+2. A Microsoft account with access to your own Subscription, or an organizational account with Owner access to the Subscription.
+3. Both `docker` and `jq` need to be installed and available on `$PATH`.
+
+
+## Cluster operations
+
+### Cluster bring-up
+
+```shell
+export KUBERNETES_PROVIDER=azure; curl -sS https://get.k8s.io | bash
+```
+
```

Comment:
You mean the `config-default.sh` specifically? I'm okay with that. The note is *very* informational, where as when it is emphasized later, it's specifically hinting at the user to take a look at it.